### ISSUE TITLE
What is the difference between F.cross_entropy() and F.nll_loss() ??

### ISSUE THREAD

**Issue by lakshmiumenon** [pytorch/pytorch#16438]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by vishwakftw** [comment]
Please consult the [forums](https://discuss.pytorch.org/) for questions like these. They are well-attended to there.

The doc string for `nn.CrossEntropyLoss()` and `nn.NLLLoss()` distinguish between the formulations quite neatly with the loss equation as well.
